src/selectors/metadata.js

```diff
@@ -24,6 +23,18 @@ const runCommandTemplates = {
   task: (name) => `kedro run --to-nodes ${name}`,
 };
 
+/**
+ * Unwrap parameters by extracting the first object
```

Comment:
I am not familiar with the object that is returned for parameters, but I wonder why are we only extracting the object of the first key of the parameter object? Is this based on any assumption of the data returned by the server? It would be good to elaborate on this assumption here in this comment. 

Also, using the term 'unwrapping' sounds a bit confusing - I would rephrase it as `returns a destructured parameter object by extracting the first object of the returned parameter metadata ( based on the assumption of the returned data from the server( or XX - insert assumption) `

Reply:
thanks @studioswong  -- i will leave the function name as unWrap because i think that sounds correct as well. 
I will mention 'because of returned data from server' for more clarity. thanks

Reply:
thanks for the follow up Rashida - I am not convinced that `unwrap` is the right word for it, this is not a term uncommonly used in codebases. What you are doing is essentially destructing the object and it would be much more intuitive / clear that way.